diff --git a/node_modules/@lucia-auth/adapter-prisma/dist/prisma.js b/node_modules/@lucia-auth/adapter-prisma/dist/prisma.js
index 64924ab..e0b0a0b 100644
--- a/node_modules/@lucia-auth/adapter-prisma/dist/prisma.js
+++ b/node_modules/@lucia-auth/adapter-prisma/dist/prisma.js
@@ -1,3 +1,11 @@
+import { Prisma } from '@prisma/client';
+
+const getObjectsWithRelationName = (inputArray) => {
+    return inputArray
+      .filter((obj) => (obj.relationName !== undefined  && !['auth_session', 'key'].includes(obj.name)))
+      .map((obj) => obj.name);
+};
+
 export const prismaAdapter = (client, modelNames) => {
     const getModels = () => {
         if (!modelNames) {
@@ -16,14 +24,23 @@ export const prismaAdapter = (client, modelNames) => {
         };
     };
     const { User, Session, Key } = getModels();
+
+    const relationalKeys = getObjectsWithRelationName(Prisma.dmmf.datamodel.models.find(model => model.name === "User").fields);
+
     return (LuciaError) => {
         return {
             getUser: async (userId) => {
-                return await User.findUnique({
+                console.log(relationalKeys);
+                const data = await User.findUnique({
                     where: {
-                        id: userId
+                        'id': userId
+                    }, 
+                    include: {
+                        ...Object.fromEntries(relationalKeys.map(key => [key, true]))
                     }
                 });
+                
+                return data
             },
             setUser: async (user, key) => {
                 if (!key) {
